<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{

	Background-color:#1B1464;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SignUp_Button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>115</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	color: Blue
}
QPushButton::Hover
{
	
	color: Grey;
}
QPushButton::pressed
{
	border: none;
	color: Blue
}</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EnterButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>115</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{

	background-color: #575fcf;
	border-bottom: 2px solid #3c40c6;
	
	border-radius: 4px;
}
QPushButton::Hover
{
	background-color : #3c40c6;
	border-bottom: 2px solid #575fcf;
	
	border-radius: 4px;
}
QPushButton::pressed
{
	background-color: #575fcf;
	border-bottom: 2px solid #3c40c6;
	
	border-radius: 4px;
}
</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterPassword">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>75</y>
      <width>391</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	
	background-color: #30336b;
	border-bottom: 4px solid #686de0;
	
	border-radius: 5px;
	
}
QLineEdit::Focus
{
	background-color: #130f40;
	border-bottom: 4px solid #4834d4;
	
	border-radius: 5px;
	color: #95afc0;
}
QLineEdit::Hover
{
	background-color: #130f40;
	border-bottom: 4px solid #4834d4;
	
	border-radius: 5px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="SignUp_Label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>I Don't have Password: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterUsername">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>391</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	
	background-color: #30336b;
	border-bottom: 4px solid #686de0;
	
	border-radius: 5px;
	
}
QLineEdit::Focus
{
	background-color: #130f40;
	border-bottom: 4px solid #4834d4;
	
	border-radius: 5px;
	color: #95afc0;
}
QLineEdit::Hover
{
	background-color: #130f40;
	border-bottom: 4px solid #4834d4;
	
	border-radius: 5px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="PassLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Your Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="UNameLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Your Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="WarningText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
